tests/integration/test_package_managers.py

```diff
@@ -161,3 +182,24 @@ def test_packages(
         )
         log.info("Compare checksums of files in source vendor folder")
         assert files_checksums == expected_files_checksums
+
+    if test_params.build_image_with_fetched_deps:
```

Comment:
I can try to break tests to some smaller separate pytests, if you want.

Reply:
Yeah I would prefer for the e2e tests to be separate

Reply:
Done